frontend/src/components/base/RenderField.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { withRouter } from 'react-router';
+import './RenderField.scss';
+
+const RenderField = ({ container = '', category = '', content = '', section = '' }) => {
```

Comment:
@cw10108 I think RenderField name is a little bit ambiguous, How about GroupInfoItem or GroupDetailItem

Reply:
we also use it at content detail container, so what about ComponentInfoItem?